mediterranean-journal-of-chemistry.csl

```diff
@@ -0,0 +1,155 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="sort-only" default-locale="en-GB">
+  <info>
+    <title>Mediterranean Journal of Chemistry</title>
+    <id>http://www.zotero.org/styles/mediterranean-journal-of-chemistry</id>
+	<link href="http://www.zotero.org/styles/mediterranean-journal-of-chemistry" rel="self"/>
+    <link href="http://www.zotero.org/styles/nature-publishing-group-vancouver" rel="template"/>
+    <link href="http://www.nature.com/tpj/TPJ_gta.pdf" rel="documentation"/>
```

Comment:
Please change this URL to: http://www.medjchem.com/index.php/medjchem/about/submissions#authorGuidelines